src/examples/graph.js

```diff
@@ -52,54 +52,63 @@ const sample: IGraph = {
   edges: [
     {
       handleText: '5',
+      tooltipText: '5',
       source: 'start1',
       target: 'a1',
       type: SPECIAL_EDGE_TYPE,
     },
     {
       handleText: '5',
+      tooltipText: '5',
```

Comment:
We should make sure that people understand this to be an optional property. If we fill in every edge with toolTipText then they might assume it's required on all edges when it's not. Perhaps we should remove the example from the below edges but keep it on the first two.